src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -9,61 +9,94 @@ import { connect } from 'react-redux';
 
 import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
 import { assetActions } from '../../data/constants/actionTypes';
-import { clearAssetsStatus, deleteAsset } from '../../data/actions/assets';
+import { clearAssetsStatus, deleteAsset, sortUpdate } from '../../data/actions/assets';
 
 export class AssetsTable extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      displayAssetsList: [],
       modalOpen: false,
       statusAlertOpen: false,
       assetToDelete: {},
+      deletedAsset: {},
+      deletedAssetIndex: null,
       elementToFocusOnModalClose: {},
-      statusAlertRef: {},
     };
 
+    this.columns = {
```

Comment:
I've moved the columns up into an instance variable instead of in the render so that I could more easily map over them an add an onSortClick in the render function.